daemon/prune.go

```diff
@@ -370,7 +373,8 @@ func (daemon *Daemon) clusterNetworksPrune(ctx context.Context, pruneFilters fil
 	for _, nw := range networks {
 		select {
 		case <-ctx.Done():
-			return rep, ctx.Err()
+			// when the context is cancelled, return current report struct and nil
```

Comment:
This is obvious from the return line, I don't think we need the comment explaining